docs/4.0/aws_oss_guide.md

```diff
@@ -0,0 +1,340 @@
+```
+
+2. Apply Updates `sudo yum update`
+3. Create a new admin user `sudo tctl users add teleport-admin ec2-user`
+```xml
+[ec2-user@ip-172-30-0-111 ~]$ sudo tctl users add teleport-admin ec2_user
+Signup token has been created and is valid for 1 hours. Share this URL with the user:
+https://teleport.acme.com:443/web/newuser/cea9871a42e780dff86528fa1b53f382
+
+NOTE: Make sure teleport.acme.com:443 points at a Teleport proxy which users can access.
+```
+![Summary for AWS Load Balancer](img/aws/teleport-admin.png)
+
+Step 5: Finish
+You've now successfully setup a simple Teleport AMI, that uses local storage and
+has itself as a node. Next we'll look at using HA services to create a more scalable 
+Teleport install. 
+
+![Summary for AWS Load Balancer](img/aws/teleport-setup.png)
+
+
+#### Reconfiguring/using a pre-existing instance
+
+To reconfigure any of this, or to do it on a running instance:
+
+1. Make the appropriate changes to /etc/teleport.d/conf
+
+* `rm -f /etc/teleport.yaml`
+
+* `systemctl restart teleport-generate-config.service`
+
+* `systemctl restart teleport-acm.service`
+
+
+If you have changed the external hostname, you may need to delete `/var/lib/teleport` and start again.
+
+
+## Deploying with CloudFormation
+We are currently working on an updated CloudFormation guide but you can start with our
+[AWS Marketplace example](https://github.com/gravitational/teleport/tree/master/assets/marketplace/cloudformation#teleport-aws-quickstart-guide-cloudformation). It requires a VPC, but 
+we expect customers to deploy within an already existing VPC. 
+
+## Deploying with Terraform
+To deploy Teleport in AWS using Terraform look at our [AWS Guide](https://github.com/gravitational/teleport/tree/master/examples/aws/terraform#terraform-based-provisioning-example-amazon-single-ami).
+
+
+### Installing Teleport to EC2 Server
+Customers run many workloads within EC2 and depending on how you work there are many 
+ways to integrate Teleport onto your servers. We recommend looking at our [Admin manual](https://gravitational.com/teleport/docs/admin-guide/#installing).
+
+In short, to add new nodes / EC2 servers that you can "SSH into" you'll need to 
+1. [Install the Teleport Binary on the Server](https://gravitational.com/teleport/docs/admin-guide/#installing)
+- [Run Teleport, we recommend using SystemD](https://gravitational.com/teleport/docs/admin-guide/#systemd-unit-file)
+- [Set the correct settings in /etc/teleport.yaml](https://gravitational.com/teleport/docs/admin-guide/#configuration-file)
+- [Add EC2 nodes to the Teleport cluster](https://gravitational.com/teleport/docs/admin-guide/#adding-nodes-to-the-cluster)
+
+## Using Teleport with EKS
+We are working on a step by step guide for working with EKS. This [blog post](https://gravitational.com/blog/teleport-aws-eks/) is a good place to start. 
+
+## Upgrading
+
+To upgrade to a newer version of Teleport:
+
+- Back up `/etc/teleport.yaml` and the contents of `/var/lib/teleport`
```

Comment:
Thinking about it, `/etc/teleport.d/` and its contents should also be backed up and copied over to the new instance.